driver/src/util.rs

```diff
@@ -104,9 +106,23 @@ pub fn batch_processing_state(
 
 pub fn create_price_finder(fee: Option<Fee>) -> Box<dyn PriceFinding> {
     let solver_env_var = env::var("LINEAR_OPTIMIZATION_SOLVER").unwrap_or_default();
+    let optimization_model_string: String =
+        env::var("OPTIMIZATION_MODEL").unwrap_or_else(|_| String::from("mip"));
+    let optimization_model;
+    match optimization_model_string.as_str() {
```

Comment:
This logic could be implemented on the struct directly. E.g. [`tryFrom<String>`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) or [`From<String>`](https://doc.rust-lang.org/std/convert/trait.From.html). 

Should the logic be case insensitive?

Reply:
I tried to implement it via the From trait. But I could not get it working :(

If I uncomment the the line 110 in utils.rs, then I am getting the following error:
```
no variant or associated item named `From` found for type `price_finding::price_finder_interface::OptimizationModel` in the current scope
   --> driver/src/util.rs:111:28
    |
111 |         OptimizationModel::From(optimization_model_string.clone());
    |                            ^^^^
    |                            |
    |                            variant or associated item not found in `price_finding::price_finder_interface::OptimizationModel`
    |                            help: there is a method with a similar name (notice the capitalization): `from`
    | 
   ::: driver/src/price_finding/price_finder_interface.rs:26:1
    |
26  | pub enum OptimizationModel {
    | -------------------------- variant or associated item `From` not found here
```